### DISCUSSION TITLE
BitOps Meeting (18 Jan 2023): Community Survey Form, CLI Ops Repo Generator, v2.4.0

### DISCUSSION THREAD

**Discussion by arm4b** [bitovi/bitops#387] [Announcements]
We invite everyone (developers, adopters, or just curious people) to join the bi-weekly BitOps Community `1 hour` Meeting on Wednesday, `18 Jan 2023, 12:00 PM US East` to discuss the ongoing project developments, updates, and plans, share use cases and stories or ask/answer questions about BitOps tool.

See https://github.com/bitovi/bitops/discussions/288 to find the meeting link and schedule.

### Updates from the previous #384 Meeting
- BitOps intro Video (@PhillypHenning) 🎉 
  - https://www.youtube.com/watch?v=BiytYu3EefY
- BitOps landing page(s) (@mickmcgrath13 @armab)
- BitOps Twitch (@PhillypHenning @LeoDiazL) 🎉
- BitOps GH Action: StackStorm

### BitOps Community Survey Form
See [BitOps Survey #304](https://github.com/bitovi/bitops/issues/304). We want to add a survey form so we can gather feedback non-stop from our potential users.

### Planning: BitOps OpsRepo Generator CLI in Golang
Discuss the repository, some first tasks, feature set, CI, tests, releases, artifacts building, publishing and distribution, how do we do the work, share the Golang knowledge, etc.

### v2.4.0 Release Planning
See https://github.com/bitovi/bitops/milestone/10 with the (5) issues that were transferred from the previous `v2.3.0`
- DeCLI? https://github.com/bitovi/bitops/issues/230
- New Cloud Provider(s)?

**Comment by mickmcgrath13** [comment]
- streams 1 per month
  - last one was to upgrade bitops in the docker-to-ec2 action

- stackstorm plan
  - good on features?
    - need to finish st2.yaml
  - blog post (by phil)
    - add the st2.yaml functionality
  - st2 community
  - linkedin GHA group

Community survey form
- form in the works (https://docs.google.com/forms/d/e/1FAIpQLSeOt-yLSNFlWFwL5uwRIuY-bqTOYY400GiMUzTysK7bg3d8pw/viewform?usp=sf_link)
- started from gpt chat 😂 
- need to add it to the landing page as part of that refactor
- need to add it to the bitops.sh docs page

New cloud providers
- more important for the next release, I think (mick)

Generator
- https://github.com/bitovi/bitops-cli
- would be nice to split tasks as small as possible so others can learn and participate
- needs good PM'ing
- next steps: phil add [his code](https://github.com/bitovi/bitops/tree/cli-tool) to the [new repo](https://github.com/bitovi/bitops-cli)
- List of TODO tasks: https://github.com/bitovi/bitops-cli

DeCLI
- didn't cover
